test/specs/commonTests.js

```diff
@@ -858,31 +858,13 @@ export const implementsImageProp = (Component, options = {}) => {
  * @param {Object} [options.requiredProps={}] Props required to render the component.
  */
 export const implementsTextAlignProp = (Component, options = {}) => {
-  const { requiredProps = {} } = options
   const { assertRequired } = commonTestHelpers('implementsTextAlignProp', Component)
 
   describe('aligned (common)', () => {
     assertRequired(Component, 'a `Component`')
 
     _noDefaultClassNameFromProp(Component, 'textAlign', options)
     _noClassNameFromBoolProps(Component, 'textAlign', options)
-
-    _.each(Component._meta.props.aligned, (propVal) => {
```

Comment:
The removed portion of the test is the useful part.  It makes the assertion that the proper `className` is added.  I see it was removed as it looped over `_meta.props.aligned`.  In fact, this test was doing nothing as the `aligned` prop name was changed to `textAlign`!

Instead of removing these assertions, however, let's loop over `SUI.TEXT_ALIGNMENTS` directly:

```jsx
_.each(SUI.TEXT_ALIGNMENTS, (propVal) => {
  ...
})
```